assets/case/reschedule_finish_hearing_deadline.js

```diff
@@ -0,0 +1,21 @@
+/* global $ */
+
+$('#finish_hearing_submit_button').on('click', function (e) {
+  const $form = $('#reschedule_hearing_form')
+  e.preventDefault()
+  // Submit data via AJAX to the form's action path.
+  $.ajax({
+    url: $form.attr('action'),
+    type: $form.attr('method'),
+    data: $form.serialize(),
+    success: function (html) {
+      if ($(html).find('.form-error-message').length) {
```

Comment:
Use `….length > 0` here to make it more clear what's going on.

Reply:
Added `.length > 0`.